app/controllers/api/v2/config_reports_controller.rb

```diff
@@ -5,7 +5,7 @@ class ConfigReportsController < V2::BaseController
       include Foreman::Controller::SmartProxyAuth
 
       before_action :find_resource, :only => %w{show destroy}
-      before_action :setup_search_options, :only => [:index, :last]
+      before_action :setup_search_options, :only => [:last]
```

Comment:
This has an interesting side affect on `api/v2/hosts/:id/config_reports`. while the reports are still correctly filtered, the `subtotal` field shows the wrong number, same as total, instead of total for the selected host